jobs/config.json

```diff
@@ -3689,6 +3689,23 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gci-gce-etcd3-sig-cli": {
```

Comment:
@krzyzacy perhaps we should drop these `etcd3`s since that's de-facto now?

Reply:
hummmm, I think the default gci-gce job is still testing against etcd2 :joy:

Reply:
maybe we should fix that too then?

Reply:
cc @msau42 @saad-ali - do we still care about etcd2 on master? :see_no_evil:

Reply:
cc @kubernetes/sig-api-machinery-pr-reviews for etcd2 questions